src/passes/SimplifyLocals.cpp

```diff
@@ -411,8 +411,8 @@ struct SimplifyLocals
     if (set && self->canSink(set)) {
       Index index = set->index;
       assert(self->sinkables.count(index) == 0);
-      self->sinkables.emplace(std::make_pair(
-        index, SinkableInfo(currp, self->getPassOptions(), features)));
+      self->sinkables.emplace(std::pair{
+        index, SinkableInfo(currp, self->getPassOptions(), features)});
```

Comment:
Could this just use `{}` rather than `std::pair{}`?

Reply:
No, It was my first attempt, but unfortunately for this case we should use `std::pair` explicitly